### DISCUSSION TITLE
Feature request for displaying file type icon instead of plain file icon for every file.

### DISCUSSION THREAD

**Discussion by a45hif** [community/community#60042] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

If possible, please add icons of each file type in Repo files like we see in code editors.

Display of files in GitHub repo:
![image](https://github.com/community/community/assets/48546357/b2bf931f-0f3e-4477-989c-bbe913d74ca5)
Simple plain icons for every file type not looks so good.

Display of files in VS Code:
![image](https://github.com/community/community/assets/48546357/acb11c22-b6bb-47fc-a49d-06507f8145a2)

**Comment by GhostOf0days** [comment]
In the meantime, I think you may like [this](https://github.com/homerchen19/github-file-icons).

**Reply by a45hif** [reply]
Ah, so we have a extension for this. Thanks!
But, I will like to see this feature in GitHub.

**Comment by rbalet** [comment]
Gitea has an implementation of it, maybe GitHub could take inspiration from it : )

https://github.com/go-gitea/gitea/pull/33837

**Comment by pansy1110** [comment]
This idea is great. I really like the Jetbrains icons. I hope it can be realized.
